numpy/polynomial/polynomial.py

```diff
@@ -7,48 +11,63 @@
 on how this module represents and works with polynomial objects is in
 the docstring for its "parent" sub-package, `numpy.polynomial`).
 
+Classes
+-------
+.. autosummary::
```

Comment:
My memory of this is that it doesn't actually work, because this rst isn't read until the first pass of autosummary, and there is no recursion so the `polynomial` entry here is never visited.

Reply:
Thanks for the heads up - it's building fine for me with sphinx version 2.4.3 though perhaps that's too new

Reply:
A bit more follow-up: I built the docs with sphinx v2.2.0, which is the minimum required in `conf.py`. The build doesn't complain and the result appears to render properly. I've published the results of running `make html` w/ sphinx 2.2.0 on commit 9bb6a51f9 to [the gh-pages of my fork](https://rossbar.github.io/numpy/reference/routines.polynomials.polynomial.html) so that the results can be more easily compared with the current docs.